app/javascript/packs/stylesheets/patient.scss

```diff
@@ -4,6 +4,10 @@
   vertical-align: top !important;
 }
 
+.patient-card-header {
+  font-size: 1.25rem;
```

Comment:
These are classes that copy the styles of `h1`s, `h2`s...etc.

again, <h-tags> are only used for hierarchy. Their actual look doesnt matter whatsoever.